src/lsps1/client.rs

```diff
@@ -236,7 +237,12 @@ where
 		}
 	}
 
-	fn request_for_info(&self, counterparty_node_id: PublicKey, channel_id: u128) {
+	/// Retrieve information from the LSP regarding the options supported.
+	///
+	/// `counterparty_node_id` is the node_id of the LSP you would like to use.
+	///
+	/// 'channel_id' is the id used to uniquely identify the channel with counterparty node.
```

Comment:
nit: Wrong ticks used here:
```suggestion
	/// `channel_id` is the id used to uniquely identify the channel with counterparty node.
```

But happy to fix this in a follow up sometime.